fvm/transactionStorageLimiter.go

```diff
@@ -71,12 +74,17 @@ func (limiter TransactionStorageLimiter) getStorageCheckAddresses(
 		addresses = append(addresses, payer)
 	}
 
+	sc := systemcontracts.SystemContractsForChain(ctx.Chain.ChainID())
 	for id := range snapshot.WriteSet {
 		address, ok := addressFromRegisterId(id)
 		if !ok {
 			continue
 		}
 
+		if limiter.shouldSkipSpecialAddress(ctx, address, sc) {
```

Comment:
what about setting a very high limit instead of skipping it?

Since the returned list is supposed to be updated registers, skipping it would look like as if the EVM address is not updated. I'm not sure if it would break any assumptions elsewhere

Reply:
This logic is only to control which accounts get their storage capacity checked. 

Having a special price/limit for storage capacity of this account might be a good solution for the future, but we have not decided what to do with the storage pricing of this account yet. Implementing that also takes more effort.

This is a temporary solution so that testing is easier in the meantime.